railib/api.py

```diff
@@ -390,40 +412,28 @@ def data(self):
         result = {
             "dbname": self.database,
             "nowait_durable": self.nowait_durable,
-            "readonly": self.readonly
+            "readonly": self.readonly,
+            # "sync_mode": "async"
```

Comment:
So the default is sync? Do we wanna default to async since this is called for the run_async case? 
also 🎉🎉🎉 I just thought of setting this flag is on the infra side, setting this here is much cleaner

Reply:
oh sorry I think the default mode is async in the backend, my bad

Reply:
Changed here https://github.com/RelationalAI/rai-sdk-python/pull/71